crates/bevy_render/src/texture/mod.rs

```diff
@@ -61,7 +61,8 @@ impl Plugin for ImagePlugin {
         app.add_plugin(RenderAssetPlugin::<Image>::with_prepare_asset_label(
             PrepareAssetLabel::PreAssetPrepare,
         ))
-        .add_asset::<Image>();
+        .add_asset::<Image>()
```

Comment:
This makes me suspicious; why not just roll register_asset_reflect into add_asset?

Reply:
What if we made `add_asset` do type registration and make a second `add_unreflected_asset` method as an escape hatch?

I'd prefer if the obvious thing to do was the correct one, rather than forcing users to remember a second step.

Reply:
I'm fine to punt on this question for now.